sudoku_solver.py

```diff
@@ -40,7 +41,9 @@ def build_cqm(x):
     for r in range(len(subsquares)):
         for c in range(len(subsquares)):
             for k in range(1,10):
-                cqm.add_constraint(quicksum(x[i][j][k-1] for i in subsquares[r] for j in subsquares[c]) == 1, label=f'subsquare {r}_{c} dig {k}')
+                cqm.add_constraint(quicksum(x[i][j][k-1] for i in subsquares[r]
+                                                for j in subsquares[c]) == 1,
```

Comment:
This is super minor, but you can line up lines 45 and 46? Using tabs will be easiest to do that.